File: semgrep-core/tests/ocaml/metavar_arg.ml
(* ERROR: *)
let foo = foo 1 2

(* ERROR: *)
let foo1 = foo a_very_long_constant_name 2

(* ERROR: *)
let foo2 = foo unsafe (* indeed *) 2

(* ERROR: *)
let foo3 = foo (bar 1 3) 2

let foo4 = foo 2 1

File: semgrep-core/tests/patterns/csharp/metavar_arg.cs
public class MetaVar
{
    public static void Main()
    {
        // ERROR:
        Foo(1, 2);

        // ERROR:
        Foo(int.MaxValue,
            2);

        // ERROR:
        Foo(int.Parse("3"), // comment
            2);

        // ERROR:
        Foo(Bar(1, 3), 2);

	// OK:
        Foo(2, 1);

	// OK:
	Foo(1, 2, 3);
    }
}
